Adyen/Analytics/AnalyticsProvider/TelemetryData.swift

```diff
@@ -0,0 +1,72 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+internal struct TelemetryData: Encodable {
+
+    // MARK: - Properties
+
+    internal var version: String? {
+        Bundle(for: AnalyticsProvider.self).infoDictionary?["CFBundleShortVersionString"] as? String
+    }
+
+    internal let channel: String = "iOS"
+
+    internal var locale: String {
+        let languageCode = Locale.current.languageCode ?? ""
+        let regionCode = Locale.current.regionCode ?? ""
+        return "\(languageCode)_\(regionCode)"
+    }
+
+    internal let userAgent: String? = nil
+
+    internal var deviceBrand: String {
+        var systemInfo = utsname()
+        uname(&systemInfo)
+        let machineMirror = Mirror(reflecting: systemInfo.machine)
+        return machineMirror.children.reduce("") { identifier, element in
+            guard let value = element.value as? Int8, value != 0 else { return identifier }
+            return identifier + String(UnicodeScalar(UInt8(value)))
+        }
+    }
```

Comment:
this also can be lazy var, the device brand doesn't change at runtime, right?

Reply:
changed to be a constant instead